README.md

```diff
@@ -80,6 +80,7 @@ documents:
 * [Installation](./docs/installation.md)
 * [Configuration](./docs/configuration.md)
 * [Supported instrumentations](./docs/supported-instrumentations.md)
+* [Migrating to OpenTelemetry .NET](./docs/migration.md)
```

Comment:
Should we link to this from the main README like proposed here, or just refer to it in contexts where it's relevant or needed?

cc @gouthamve